cmd/controller/certificates.go

```diff
@@ -232,6 +237,36 @@ func (c *Certificates) Init(ctx context.Context) error {
 	return eg.Wait()
 }
 
+func detectLocalIPs() ([]string, error) {
+	var localIPs []string
+	ifaces, err := net.Interfaces()
+	if err != nil {
+		return nil, err
+	}
+	for _, i := range ifaces {
+		addrs, err := i.Addrs()
+		if err != nil {
+			return nil, err
+		}
+
+		for _, addr := range addrs {
+			switch v := addr.(type) {
+			case *net.IPNet:
+				if v.IP.To4() == nil {
+					continue
+				}
+				localIPs = append(localIPs, v.IP.To4().String())
+			case *net.IPAddr:
+				if v.IP.To4() == nil {
+					continue
+				}
+				localIPs = append(localIPs, v.IP.To4().String())
```

Comment:
Isn't the linked issue just about localhost? Maybe we should filter for everything in the 127.0.0.0/8 subnet here?

Reply:
Right, will fix